src/argilla/client/feedback/dataset/remote/base.py

```diff
@@ -102,7 +105,9 @@ def __len__(self) -> int:
         return response.parsed.records.count
 
     @allowed_for_roles(roles=[UserRole.owner, UserRole.admin])
-    def __getitem__(self, key: Union[slice, int]) -> Union[RemoteFeedbackRecord, List[RemoteFeedbackRecord]]:
+    def _get_records(
+        self, key: Union[slice, int], filters: Dict[str, Any] = {}
+    ) -> Union[RemoteFeedbackRecord, List[RemoteFeedbackRecord]]:
```

Comment:
```suggestion
    ) -> Union[RemoteFeedbackRecordBase, List[RemoteFeedbackRecordBase]]:
```

Reply:
There's no `RemoteFeedbackRecordBase` and the `RemoteFeedbackRecord` is the same on both the `RemoteFeedbackDataset` and the `FilteredRemoteFeedbackDataset` as we want to keep the record-level operations (update and delete)

Reply:
ups sorry. I saw an extra `s`